tools/Dockerfile

```diff
@@ -0,0 +1,15 @@
+
+FROM        ocaml/opam2:debian-9-ocaml-4.06
+MAINTAINER  Christian Lindig <christian.lindig@citrix.com>
+
+RUN sudo apt-get install -qq -yy \
+  debianutils hwdata libdlm-dev libffi-dev libgmp-dev libnl-3-200   \
+  libnl-route-3-200 libpam-dev libpci-dev libssl-dev libsystemd-dev \
+  libxen-dev linux-libc-dev m4 perl pkg-config python uuid-dev
+
+COPY . /mnt
+
+RUN opam repo remove --all default
+RUN opam repo add xs-opam file:///mnt
+RUN opam depext -y xs-toolstack
```

Comment:
These 4 could also be on the same line, if anything changes in `.` then docker will need to recreate every layer from the COPY line, so the ones below it wouldn't be cached, and having it on a single line could mean a smaller image.
If we want we could also rm opam's cache like done in the spec file to make the container smaller.